### PR TITLE
Updating variables pages with icons, screenshots and amendments

### PR DESCRIPTION
**Pull request by felixi-octopus**
[sc-84718]
# Background
Updating Project Variables docs as part of #projects-docs-refresh.

# Results
* Updated screenshots, last modified dates, page icons
* Added older version sections
* Minor fixes and amendments

### CODE REVIEW

File: src/pages/docs/projects/variables/sensitive-variables.md [hunk 1]
```diff
@@ -94,14 +95,14 @@ You could also make `DB.Username` or any of the other components of this templat
...
 - **Avoid treating entire files as sensitive**: Imagine you're consuming a YAML file from a variable, and that YAML file contains a secret. Rather than treating the whole YAML file as sensitive, you should create two variables: one sensitive variable containing just the secret, and one non-sensitive variable for the YAML file which uses [variable substitution](/docs/projects/variables/variable-substitutions) to substitute in the sensitive variable. This gives Octopus a much tighter scope when looking for sensitive variables to mask.
-- **Avoid sequences that are part of the variable substitution syntax** {#substitution-syntax-mistakes}: For example, the sequence `##{` will be replaced by `#{` by logic that's part of [referencing variables](/docs/projects/variables/#use-variables-in-step-definitions) so you would need to escape it by modifying it to be `###{` which will result in `##{`, see also [variable substitution syntax](/docs/projects/variables/variable-substitutions).
```
Comment (felixi-octopus) [inline src/pages/docs/projects/variables/sensitive-variables.md:104]: `{#substitution-syntax-mistakes}` didn't work and just displayed the anchor as is

File: src/pages/docs/projects/variables/system-variables.md [hunk 2]
```diff
@@ -1077,7 +1078,7 @@ Example: *True*
 
 `Octopus.Action.Azure.DeploymentLabel`
 
-If set, the custom deployment label will be used for the Azure cloud service deployment. **Introduced in Calamari version 3.4.1.**.
```
Comment (felixi-octopus) [inline src/pages/docs/projects/variables/system-variables.md:1080]: This version would be quite old now and this variable is practically in all modern versions of Octopus Server. I don't think it's worth tracking down and noting which version of Server this corresponds to.

File: src/pages/docs/projects/variables/sensitive-variables.md (hunk 1 above)
Comment (steve-fenton-octopus) [reply src/pages/docs/projects/variables/sensitive-variables.md:104]: Hey @felixi-octopus do you want this to be the target for a bookmark, or to link to a bookmark elsewhere?

Comment (felixi-octopus) [reply src/pages/docs/projects/variables/sensitive-variables.md:104]: It was originally meant to be the target for a link in the `Sensitive variables stored in Octopus` paragraph but I've removed both this anchor and the link as they're kind of redundant - the `Sensitive variables stored in Octopus` paragraph already contains another link to this paragraph's heading

**Review by steve-fenton-octopus — APPROVED** [review]
LGTM!

**Pull request merged**